presto-main/src/main/java/com/facebook/presto/security/AccessControlManager.java

```diff
@@ -150,6 +150,15 @@ public void checkCanSetUser(Optional<Principal> principal, String userName)
         authenticationCheck(() -> systemAccessControl.get().checkCanSetUser(principal, userName));
     }
 
+    @Override
+    public void checkQueryIntegrity(Identity identity, String query)
+    {
+        requireNonNull(identity, "extraCredentials is null");
```

Comment:
@mayankgarg1990 Error message should be "identity is null", not "extraCredentials is null"

Reply:
@mbasmanova - good catch :) Fixed it.